### ISSUE TITLE
[a11y] Define list of supported accessibility tools

### ISSUE THREAD

**Issue by data-sync-user** [mozilla-mobile/mozilla-vpn-client#5428]
We should define which native (and/or third party?) accessibility tools or features we support. This list should be added to the repository as a readme. 

An incomplete list to get us started:

Defaults:

* Android: [+TalkBack+](https://support.google.com/accessibility/android/answer/6283677?hl=en)
* iOS: [+VoiceOver+](https://support.apple.com/guide/iphone/turn-on-and-practice-voiceover-iph3e2e415f/ios)
* Macos: [+VoiceOver+](https://www.apple.com/voiceover/info/guide/_1121.html)
* Windows:[ ﹍Narrator﹍|https://support.microsoft.com/en-us/windows/chapter-1-introducing-narrator-7fe8fd72-541f-4536-7658-bfc37ddaf9c6]
* Linux: :question: [+Orca+](https://help.ubuntu.com/stable/ubuntu-help/a11y-screen-reader.html.en)

> What else should be here?

┆Issue is synchronized with this [Jira Task](https://mozilla-hub.atlassian.net/browse/VPN-3659)
┆Reporter: Lesley Norton
